specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2021-09-09/examples/ImportImageFromPublicRegistry.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2021-06-01-preview",
```

Comment:
@AntonioVT please update this to use the 2021-09 API version

Reply:
Done